### PR TITLE
Redirect PPro

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/0/1206957384492835/f

### Description
See task

### Steps to test this PR
See task

### CODE REVIEW

**Comment by marcosholgado** [comment]
* **#4381** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4381?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @marcosholgado and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by aitorvs — APPROVED** [review]
Left couple comments to fix, but LGTM.

Was not able to manually test it because I don't have US account though.

File: subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelSender.kt [hunk 1]
```diff
@@ -194,6 +195,9 @@ class SubscriptionPixelSenderImpl @Inject constructor(
     override fun reportAddEmailSuccess() =
         fire(SUBSCRIPTION_ADD_EMAIL_SUCCESS)
 
+    override fun reportPrivacyProRedirect() =
+        fire(SUBSCRIPTION_ADD_EMAIL_SUCCESS)
```
Comment (karlenDimla) [inline subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelSender.kt:199]: @marcosholgado  should be SUBSCRIPTION_PRIVACY_PRO_PATH (odd naming too) right?

**Review by karlenDimla — APPROVED** [review]
Tested and worked as expected. (1 minor correction on pixel)

**Review by lmac012 — APPROVED** [review]
Works as expected 👍

File: subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelSender.kt (hunk 1 above)
Comment (marcosholgado) [reply subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelSender.kt:199]: Yup, changed the name of the pixel too.

**Review by aitorvs — APPROVED** [review]

**Pull request merged**